### PR TITLE
#13 registration new user

### PR DESCRIPTION
**Pull request by IgorNazaryok**

### CODE REVIEW

**Review by horba — CHANGES_REQUESTED** [review]
please, use vuex to store and manage data

File: front/src/App.vue [hunk 1]
```diff
@@ -1,19 +1,191 @@
...
+          <p>Аккаунт создан!</p>
+          <p>Ты стал частью большого мира, где каждый может внести свою
+            лепту в то, чтобы процесс поиска фильма, сериала или мультфильма был проще и
+            увлекательней.
+          </p>
+          <p>Вперед за просмотрами, оценками и комментариями!</p>
+        </div>
+      </div>
+    </div>
+    <a href="#registration">
```
Comment (horba) [inline front/src/App.vue:62]: why do we need this href?

File: front/src/App.vue [hunk 2]
```diff
@@ -1,19 +1,191 @@
 <template>
-  <div id="app">
+  <div id="app" class="font">
     <div>
       <router-link to="/">Home</router-link>
     </div>
     <router-view />
+    <div id="registration" class="blackout">
```
Comment (horba) [inline front/src/App.vue:7]: please provide separate component for registration modal

File: front/src/App.vue [hunk 3]
```diff
@@ -1,19 +1,191 @@
...
+        },
+        body: JSON.stringify({
+          UserName: this.ruleForm.userName,
+          Email: this.ruleForm.email,
+          UserPassword: this.ruleForm.password
+        })
+      })
+        .then(response => response.text())
+        .then(data => {
+          if (data === '1') {
```
Comment (horba) [inline front/src/App.vue:105]: what is it 1 means?

File: front/src/App.vue [hunk 4]
```diff
@@ -1,19 +1,191 @@
...
+            this.ruleForm.userName = this.ruleForm.email = this.ruleForm.password = '';
+            document.location.href = '#created';
+          } else {
+            alert('Данный email уже зарегистрирован');
```
Comment (horba) [inline front/src/App.vue:109]: i think we have in element-ui more ux friendly  components

File: front/src/App.vue [hunk 5]
```diff
@@ -1,19 +1,191 @@
...
+.blackout {
+  background: rgba(102, 102, 102, 0.5);
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: none;
+}
+.okno {
```
Comment (horba) [inline front/src/App.vue:127]: is it about window? no russion words, please. You might return to article in wiki about ninja code

**Review by horba — CHANGES_REQUESTED** [review]

File: front/src/App.vue (hunk 1 above)
Comment (IgorNazaryok) [reply front/src/App.vue:62]: > why do we need this href?

link in the form of a button to go to registration

File: front/src/App.vue (hunk 2 above)
Comment (IgorNazaryok) [reply front/src/App.vue:7]: > please provide separate component for registration modal
done

File: front/src/App.vue (hunk 3 above)
Comment (IgorNazaryok) [reply front/src/App.vue:105]: done

File: front/src/App.vue (hunk 4 above)
Comment (IgorNazaryok) [reply front/src/App.vue:109]: done

File: front/src/App.vue (hunk 5 above)
Comment (IgorNazaryok) [reply front/src/App.vue:127]: done

**Pull request closed without merging**